src/rocksdb.rs

```diff
@@ -513,6 +513,81 @@ impl DB {
         self.get_cf_opt(cf, key, &ReadOptions::new())
     }
 
+    pub fn multi_get_cf_opt(&self,
+                            cf: &CFHandle,
+                            keys_vec: Vec<&[u8]>,
```

Comment:
&[Vec<u8>] is kind of strange. we've already use &[u8] to denote Slice.
I think `&[&[u8]]` is ok.
@siddontang